tools/flang2/flang2exe/upper.cpp

```diff
@@ -6500,6 +6500,12 @@ lookup_modvar_alias(SPTR sptr)
   return NULL;
 }
 
+SPTR
+get_symbol_start(void)
+{
+  return (SPTR) (oldsymbolcount +1);
```

Comment:
What is this magic function doing?
Free casting.

Reply:
This is to get the start of symbols of current routine.

Reply:
OK. Please document what this is doing someplace. Your explanation and the name of the function don't seem perfectly aligned in my reading.

By the way, this seems like it should already be "a thing" someplace. Is this a duplicate of another function or macro?

Note on free-casting of types: we discourage the use of casts like this because we'd rather that people use well-defined access methods, defined properly in one place, rather than write (TYPE) (variable + int-const) all over the code base. It's a large engineering cost (and bug multiplier) to fix the latter should something change.